brig/cmd/brig/commands/build_logs.go

```diff
@@ -22,11 +22,13 @@ build.
 )
 
 var (
+	logsInit bool
 	logsJobs bool
 	logsLast bool
 )
 
 func init() {
+	buildLogs.Flags().BoolVarP(&logsInit, "init", "i", false, "Show init container logs as well as the worker log")
```

Comment:
I'm wondering whether we should enable both `--init` and `--jobs` by default.

Reply:
For `--init` I'm thinking that enabling it by default makes sense, as that's the only way to grab logs from vcs-sidecar, AND you have no way to access it from within worker/brigade.js.

For `--jobs`, it's really useful for debugging jobs. But usually any job failure should be handled correctly by `brigade.js` with appropriate error logs to `console`. With correctly authored `brigade.js`, `--jobs` by default may result in too verbose logs.

Reply:
That said - Please let me defer this to https://github.com/brigadecore/brigade/issues/938 for further discussion!

Reply:
Absolutely, we shouldn't keep this PR from getting merged because of this.